<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>103</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(252, 175, 62);
font:bold;</string>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="iniciar1">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>103</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Aharoni CLM&quot;;
font:bold;</string>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>103</y>
      <width>34</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:bold;
font-size:18px;
</string>
    </property>
    <property name="text">
     <string>C++</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>136</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:bold;
font-size:18px;
</string>
    </property>
    <property name="text">
     <string>COURSERA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iniciar2">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>136</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Aharoni CLM&quot;;
font:bold;</string>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_2">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>136</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(252, 175, 62);
font:bold;</string>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_3">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>169</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(252, 175, 62);
font:bold;</string>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>169</y>
      <width>84</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:bold;
font-size:18px;
</string>
    </property>
    <property name="text">
     <string>ANDROID</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iniciar3">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>169</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Aharoni CLM&quot;;
font:bold;</string>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="iniciar4">
    <property name="geometry">
     <rect>
      <x>301</x>
      <y>202</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);
color: rgb(255, 255, 255);
font: 75 11pt &quot;Aharoni CLM&quot;;
font:bold;</string>
    </property>
    <property name="text">
     <string>INICIAR</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_4">
    <property name="geometry">
     <rect>
      <x>112</x>
      <y>202</y>
      <width>171</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
selection-background-color: rgb(252, 175, 62);
font:bold;</string>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>202</y>
      <width>39</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:bold;
font-size:18px;
</string>
    </property>
    <property name="text">
     <string>POO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>266</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:bold;
font-size:18px;
center;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#204a87;&quot;&gt;GESTOR DE ACTIVIDAD DIARIA &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; color:#204a87;&quot;&gt;(3 HORAS)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="gen">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>370</y>
      <width>301</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-color: rgb(239, 41, 41);</string>
    </property>
    <property name="maximum">
     <number>15</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="estado">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>53</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Inactivo</string>
    </property>
   </widget>
   <widget class="QLabel" name="actividad">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>260</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#204a87;&quot;&gt;TOTAL HOY&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>301</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>71</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:3px;
border-color: red;
background-color:red;
color:white;
font:bold;</string>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
